bytes.go

```diff
@@ -40,6 +47,25 @@ func (b Base2Bytes) String() string {
 	return ToString(int64(b), 1024, "iB", "B")
 }
 
+// UnmarshalTOML implements TOML unmarshal interface.
+func (b *Base2Bytes) UnmarshalTOML(value interface{}) error {
+	var err error
+
+	switch v := value.(type) {
+	case string:
+		*b, err = ParseBase2Bytes(v)
+		if err != nil {
+			return err
+		}
+	case int, int8, int16, int32, int64:
+		*b = Base2Bytes(reflect.ValueOf(v).Int())
```

Comment:
I think it can fail with an overflow error if v > MaxInt32. Could you please add a test for int64 value that is > max int32

Reply:
Also I'd prefer not to use reflect unless necessary, which it doesn't appear to be.

Reply:
Updated. Thank you for review. 

Talking about overflow. The following code in parser checks for overflows: 

```go
			if e, ok := err.(*strconv.NumError); ok &&
				e.Err == strconv.ErrRange {

				p.panicf("Integer '%s' is out of the range of 64-bit "+
					"signed integers.", it.val)
```

https://github.com/BurntSushi/toml/blob/a368813c5e648fee92e5f6c30e3944ff9d5e8895/parse.go#L196

Also in TOML spec ints are always int64.